packages/instrumentation-amqplib/package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "opentelemetry-instrumentation-amqplib",
+    "version": "0.2.2",
+    "description": "open telemetry instrumentation for the `amqplib` package",
+    "keywords": [
+        "amqplib",
+        "opentelemetry",
+        "rabbitmq",
+        "AMQP 0-9-1"
+    ],
+    "homepage": "https://github.com/aspecto-io/opentelemetry-amqplib",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "files": [
+        "dist/src/**/*.js",
+        "dist/src/**/*.d.ts",
+        "LICENSE",
+        "README.md"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aspecto-io/opentelemetry-amqplib.git"
+    },
+    "scripts": {
+        "build": "tsc",
+        "prepare": "yarn run build",
+        "test": "mocha",
+        "watch": "tsc -w",
+        "version:update": "node ../../scripts/version-update.js",
+        "version": "yarn run version:update",
+        "test-all-versions": "tav",
+        "test:ci": "yarn test-all-versions",
+        "test:docker:run": "docker run -d --hostname demo-amqplib-rabbit --name amqplib-unittests -p 22221:5672 rabbitmq:3"
+    },
+    "bugs": {
+        "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
+    },
+    "dependencies": {
+        "@opentelemetry/api": "^0.17.0",
+        "@opentelemetry/instrumentation": "^0.17.0",
+        "@opentelemetry/semantic-conventions": "^0.17.0",
+        "test-all-versions": "^5.0.1"
```

Comment:
dev dependency

Reply:
good catch, thanks :)
Move it to the right place